registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/eventbus/KafkaMosipEventBus.java

```diff
@@ -443,6 +446,8 @@ public void consumerHealthCheck(Handler<HealthCheckDTO> eventHandler,
 				healthCheckDTO.setEventBusConnected(false);
 				healthCheckDTO.setFailureReason(f.cause().getMessage());
 			}
+			logger.info("Consumer health check ended {} {}",
+					address + " end " + DateUtils.formatToISOString(DateUtils.getUTCCurrentDateTime()));

```

Comment:
we cannot understand the releation between the above 2 logs, we have to add the timestamp to a variable and use it in both places to make the corelation

Reply:
fixed